scripts/cloudhealth/add_billing_role.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+# Copyright 2017 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.""
+
+'''
+This script adds a role to AWS accounts to enable access from cloudhealth
+'''
+
+from ConfigParser import ConfigParser
+
+import argparse
+import json
+import logging
+import os
+import sys
+
+import boto3
+import botocore.exceptions
+import requests.exceptions
+
+from cloudhealth import CloudHealthAwsAccount
+
+LOG_FORMAT = '%(asctime)-15s [%(levelname)s] (%(name)s.%(funcName)s) %(message)s'
+AWS_CREDENTIALS = os.environ['HOME'] + '/.aws/credentials'
```

Comment:
Is this needed?  This is the default AWS_CREDENTIALS value and if someone has altered it, it may be for good reason.

Reply:
It was in an earlier version of the script. Looks like it is no longer needed.